src/algorithms/calorimetry/CalorimeterHitsMerger.cc

```diff
@@ -38,19 +38,14 @@ void CalorimeterHitsMerger::initialize() {
     m_log->info(fmt::format("ID mask in {:s}: {:#064b}", m_readout, id_mask));
 }
 
-void CalorimeterHitsMerger::execute() {
+std::unique_ptr<edm4eic::CalorimeterHitCollection> CalorimeterHitsMerger::execute(const edm4eic::CalorimeterHitCollection &input) {
+    auto output = std::make_unique<edm4eic::CalorimeterHitCollection>();
 
     // find the hits that belong to the same group (for merging)
     std::unordered_map<long long, std::vector<const edm4eic::CalorimeterHit *>> merge_map;
```

Comment:
Explicit `int64_t` here too? Or `uint64_t` as used below.

Reply:
How about `decltype`, since this is a masked `cellID`?

Reply:
I'm not sure what would be an argument of said decltype. If you know what it is, I'm happy to add. Had to settle for `uint64_t` for now.